adminSiteServer/mockSiteRouter.tsx

```diff
@@ -379,16 +378,6 @@ mockSiteRouter.get("/headerMenu.json", async (req, res) => {
     res.send(await renderMenuJson())
 })
 
-mockSiteRouter.use(
-    // Not all /app/uploads paths are going through formatting
-    // and being rewritten as /uploads. E.g. blog index images paths
-    // on front page.
-    ["/uploads", "/app/uploads"],
-    express.static(path.join(WORDPRESS_DIR, "web/app/uploads"), {
-        fallthrough: false,
-    })
-)
-
```

Comment:
images on the remaining wordpress pages will not render now that they are not available locally, so we should probably redirect to R2 (or document it so we know it is a wontfix)

Reply:
Good catch - I added a new route that redirects to assets.ourworldindata.org for wp images